components/Launcher.vue

```diff
@@ -8,26 +8,61 @@
         align="center"
       >
         <h4 class="pb-3">Please complete the recaptcha to launch the app</h4>
-        <Recaptcha />
+        <vue-recaptcha
```

Comment:
I think they're has been some new changes here, keep the ones that are on the next branch